### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by KritikaSachdeva**

### CODE REVIEW

File: client/src/app/project/project.component.scss [hunk 1]
```diff
@@ -1,95 +1,113 @@
...
-    line-height: 2.5;
-    background-color: #6C7A8D;
-    color: #C3CEDD;
-    height: 55px;
-  }
-
-  .panel-info {
-    border: 0;
-  }
+  // .user-list {
```
Comment (abharana24) [inline client/src/app/project/project.component.scss]: remove commented code.

File: client/src/app/admindashboard/admindashboard.component.ts [hunk 2]
```diff
@@ -40,9 +39,6 @@ export class AdmindashboardComponent implements OnInit, OnChanges {
       submenus: [
         {
           title: "Add New Project"
```
Comment (abharana24) [inline client/src/app/admindashboard/admindashboard.component.ts]: add show all projects here

Comment (KritikaSachdeva) [reply client/src/app/admindashboard/admindashboard.component.ts]: done

File: client/src/app/project/project.component.scss (hunk 1 above)
Comment (KritikaSachdeva) [reply client/src/app/project/project.component.scss]: done

**Review by abharana24 — APPROVED** [review]

**Pull request merged**